cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-list-header.tsx

```diff
@@ -1,22 +1,21 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2021 Intel Corporation
 //
 // SPDX-License-Identifier: MIT
 
-import React from 'react';
-import { Row, Col } from 'antd/lib/grid';
 import {
-    LockFilled,
-    UnlockOutlined,
-    EyeInvisibleFilled,
-    EyeOutlined,
     CaretDownOutlined,
     CaretUpFilled,
+    EyeInvisibleFilled,
+    EyeOutlined,
+    LockFilled,
+    UnlockOutlined,
 } from '@ant-design/icons';
+import { Col, Row } from 'antd/lib/grid';
 import Tooltip from 'antd/lib/tooltip';
-
-import AnnotationsFiltersInput from 'components/annotation-page/annotations-filters-input';
 import StatesOrderingSelector from 'components/annotation-page/standard-workspace/objects-side-bar/states-ordering-selector';
+import React from 'react';
```

Comment:
Please, do 3rdparty import before internal import
What is a reason to change import order in this file?